### PR TITLE
[IMPROVEMENT][LAYOUT] Profile Layout according to a web interface

### LINKED ISSUE #2079: [IMPROVEMENT] Make Fragment Profile layout more like the default elements
## Description
The current profile fragment its different from other instances of Rocket.Chat
- The is no text on top of each EditText 
- EditText border is a little darker

#### WEB
image

#### ANDROID
image2

### PR DESCRIPTION
**Pull request by arthurbdiniz**
@RocketChat/android

Closes #2079

#### Screenshots or GIF for the change:
app

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -90,11 +90,14 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <string name="msg_check_this_out">Check this out</string>
     <string name="msg_share_using">Share using</string>
     <string name="msg_profile_update_successfully">Profile update successfully</string>
-    <string name="msg_username">username</string>
+    <string name="msg_username">Username</string>
+    <string name="msg_profile_username_hint" translatable="false">Please enter your username…</string>
```
Comment (philipbrito) [inline app/src/main/res/values/strings.xml]: Don't declare a translatable string as not translatable, please.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -90,11 +90,14 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
...
     <string name="msg_username_or_email">Username or email</string>
     <string name="msg_password">Password</string>
     <string name="msg_name">Name</string>
+    <string name="msg_profile_name_hint" translatable="false">Please enter your name…</string>
```
Comment (philipbrito) [inline app/src/main/res/values/strings.xml:122]: Don't declare a translatable string as not translatable, please.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -90,11 +90,14 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
...
     <string name="msg_email">Email</string>
+    <string name="msg_profile_email_hint" translatable="false">Please enter your email address…</string>
```
Comment (philipbrito) [inline app/src/main/res/values/strings.xml]: Don't declare a translatable string as not translatable, please.

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -113,12 +113,20 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
...
     <string name="msg_check_this_out">Check this out</string>
     <string name="msg_share_using">Share using</string>
+<<<<<<< HEAD
+    <string name="msg_profile_update_successfully">Profile update successfully</string>
+    <string name="msg_username">Username</string>
+    <string name="msg_profile_username_hint" translatable="false">Please enter your username…</string>
+=======
     <string name="msg_profile_updated_successfully">Profile updated successfully</string>
     <string name="msg_username">username</string>
+>>>>>>> dd938089e9ef495da5a5afd57651f611c1ad7bae
```
Comment (philipbrito) [inline app/src/main/res/values/strings.xml]: ?

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Review by philipbrito — APPROVED** [review]
I've fixed the requested changes here.
Thanks for sending this PR.

File: app/src/main/res/values/styles.xml [hunk 5]
```diff
@@ -262,7 +262,18 @@
...
+        <item name="android:layout_height">48dp</item>
+        <item name="android:layout_marginTop">8dp</item>
+    </style>
+
+    <style name="Profile.TextView" parent="TextAppearance.AppCompat.Title">
+        <item name="android:textSize">16sp</item>
+        <item name="android:fontFamily">sans-serif-medium</item>
+        <item name="android:textStyle">normal</item>
+        <item name="android:layout_marginTop">24dp</item>
+        <item name="android:layout_gravity">left</item>
```
Comment (hfeky) [inline app/src/main/res/values/styles.xml:274]: `layout_gravity` should be `start` instead of `left` to support RTL languages.

Comment (philipbrito) [reply app/src/main/res/values/styles.xml:274]: Thanks for the catch @HusseinElFeky!

Changed!
https://github.com/RocketChat/Rocket.Chat.Android/commit/280f1c1ceca560f20f6045acc80f051b2a151e80

**Pull request merged**